apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-examples/platform-dynamic-page-non-collapsible-example.component.html

```diff
@@ -0,0 +1,155 @@
+<button fd-button (click)="openPage()">Click to open full screen</button>
+<div cdkScrollable class="overlay" #overlay>
+    <ng-container *ngIf="fullscreen">
+        <div class="overlay-content">
+            <fdp-dynamic-page background="transparent" size="large" ariaLabel="Example Dynamic Page">
+                <fdp-dynamic-page-title
+                    title="Balenciaga Tripple S Trainers"
+                    subtitle="Oversized multimaterial sneakers with quilted effect"
+                >
+                    <!-- breadcrumb content -->
+                    <fd-breadcrumb>
+                        <fd-breadcrumb-item>
+                            <a fd-breadcrumb-link [attr.href]="'#'">Men</a>
```

Comment:
why do you need here binding?
can we use like this `href="#"`

Reply:
done